src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -23,34 +23,34 @@ You can customize the creation of a new browser context using the following opti
+| - | ------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span title="Not implemented">❌</span> | acceptDownloads             | boolean | Whether to automatically download files. Defaults to `true`. Follow the progress in [issue #98](https://github.com/grafana/xk6-browser/issues/98). |
+| <span title="Not implemented">❌</span> | baseURL                     | string  | The base URL to use for all relative URLs. For example: `/open-source/` will be converted to `https://k6.io/open-source/` if `baseURL` is set to `https://k6.io`. Follow the progress in [issue #432](https://github.com/grafana/xk6-browser/issues/432). |
+| ✅ | bypassCSP                                                           | boolean | Whether to bypass a page's Content-Security-Policy. |
+| ✅ | colorScheme                                                         | string  | Whether to display a page in dark or light mode by emulating the 'prefers-colors-scheme' media feature. It can be one of `'light'`, `'dark'`, `'no-preference'`. See [page.emulateMedia](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options and the [example](https://github.com/grafana/xk6-browser/blob/main/examples/colorscheme.js). Defaults to `'light'`. |
+| <span title="Work in progress">🚧</span> | deviceScaleFactor          | number  | Sets the resolution ratio in physical pixels to the resolution in CSS pixels. See the [link](https://github.com/grafana/xk6-browser/blob/main/examples/device_emulation.js) as an example. Defaults to `1`. Follow the progress in [issue #433](https://github.com/grafana/xk6-browser/issues/433). |
+| ✅ | extraHTTPHeaders                                                    | object  | Contains additional HTTP headers to be sent with every request, where the keys are HTTP headers and values are HTTP header values.                                                                                                                                                                                                                                                       |
+| <span title="Not implemented">❌</span> | forcedColors                | string  | Enforces a limited color palette on the page by emulating `forced-colors` media feature. It can be one of `'active'` or `'none'`. Defaults to `'none'`.                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| <span title="Not implemented">❌</span> | forcedColors                | string  | Enforces a limited color palette on the page by emulating the `forced-colors` media feature. It can be one of `'active'` or `'none'`. Defaults to `'none'`.                                                                                                                                                                                                                                  |
```